lib/mi_bridges/driver/job_income_more_about_page.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module MiBridges
+  class Driver
+    class JobIncomeMoreAboutPage < BasePage
+      TITLE = /More About .*'s Job/
+
+      delegate :members, to: :snap_application
+
+      def setup; end
+
+      def fill_in_required_fields
+        fill_in "Name of Employer", with: current_member.employed_employer_name
+        select current_member.employed_pay_interval,
+          from: "How often does Complete (40) get paid? "\
+                "This is Complete (40)'s pay period"
+
+        if hourly?
+          fill_in_hourly
+        else
+          fill_in_salary
+        end
+      end
+
+      def continue
+        click_on "Next"
+      end
+
+      private
+
+      def current_member
+        title = find("h1").text
+        name = title.scan(/More About (.*)'s Job/).flatten.first
+        members.select do |member|
+          member.mi_bridges_formatted_name == name
+        end.first
+      end
+
+      def hourly?
+        current_member.employed_pay_interval == "Hourly"
+      end
+
+      def fill_in_hourly
+        fill_in "If Complete (40) gets paid by the hour, ",
```

Comment:
I think we need to replace `Complete (40)` with the `current_member.mi_bridges_formatted_name` in this file

Reply:
Ugh. Brain-fart